### ISSUE TITLE
Can't resolve `universal-cookie` from within `react-cookie`

### ISSUE THREAD

**Issue by nflatley-zengenti** [bendotcodes/cookies#705]
I am working on a React project that is using webpack 4 and babel for transpiling to cjs and esm targets.

Since we upgraded our react-cookie and universal-cookies to v7 we cannot build our project.

```
 ERROR  Failed to compile with 1 errors                                                                          11:31:17

This dependency was not found:

* universal-cookie in ./node_modules/react-cookie/esm/index.mjs

To install it, you can run: npm install --save universal-cookie
× ｢wdm｣: 
ERROR in ./node_modules/react-cookie/esm/index.mjs
Module not found: Error: Can't resolve 'universal-cookie' in 'C:\dev\website\node_modules\react-cookie\esm'
 @ ./node_modules/react-cookie/esm/index.mjs 8:0-39 9:0-54 9:0-54 12:47-54 23:25-32
```
Tracing why this might be happening I noticed the "main" attribute in the `universal-cookie` `package.json` is missing

I added it back in into my local node_modules and now the project builds successfully

![image](https://github.com/user-attachments/assets/45c9fa6c-8248-48f3-8d86-e4351e1a4041)

I can fork and PR to fix this if someone here is happy to review and release.

Many thanks
Neil

**Comment by nflatley-zengenti** [comment]
It looks like the main attribute got lost unintentionally here:
https://github.com/bendotcodes/cookies/pull/441/files#diff-6bcef427012893dd46647eaa63736e15ec8f7fe5b3677ac19dfd6854e2089848L5

**Comment by eXon** [comment]
Fixed in 7.2.2

**Comment by nflatley-zengenti** [comment]
I've updated to the new release and the problem is resolved. Thanks very much!
